README.md

```diff
@@ -2,6 +2,54 @@
 
 A collection of high-quality purpose-built IoT reference applications.
 
-1. [**Indoor Floor-Level Tracker**](./indoor-floor-level-tracker/): A highly accurate floor-level
-tracker for response teams at emergency events.
-2. [**Fridge Fleet Monitor**](./fridge-fleet-monitor/): A LoRa-based temperature, humidity, and door open/close state monitoring system for a fleet of refrigerators.
+## What is this?
+
+At Blues our attempt is to make wireless IoT easier for developers and more affordable for all. This list of app accelerators is intended to help you get new IoT projects up and running quickly.
+
+Each accelerator:
+
+* 📚 **Provides a complete walkthrough.**
+  * By following the writeups you can recreate each accelerator in its entirety.
+* ⚙️ **Includes a full list of the hardware you need.**
+  * We link to specific sensors and boards so you know exactly what to get.
+* 👨‍💻👩‍💻 **Uses open-source firmware.**
+  * We provide the firmware as well as instructions for running it on your devices.
+* 📱 **Provides a way of viewing your device data.**
+  * We show you how to get the data to a useful place to view, which sometimes is a web dashboard, and sometimes is SMS alerts.
+* 📘 **Is free to copy, fork, and build upon.**
+  * This repository uses the Apache 2.0 license, which means each accelerator is free for personal and commercial use.
+
+## The Accelerators
+
+### 1: 🏢 Indoor Floor-Level Tracker
```

Comment:
Change the : to a . - the other items in the list have periods behind their numbers

Reply:
👍